scaleway/loadbalancers.go

```diff
@@ -375,6 +378,11 @@ func (l *loadbalancers) GetLoadBalancerName(ctx context.Context, clusterName str
 func extractNodesExternalIps(nodes []*v1.Node) []string {
 	var nodesList []string
 	for _, node := range nodes {
+		if address, ok := node.Annotations[nodeAnnotationNodePublicIP]; ok {
```

Comment:
Shouldn't we fallback to internal IP if external IP does not exist?

Reply:
Not a big fan ATM, will depend of VPC, ipv6, etc...